app/terminal/BUILD

```diff
@@ -1,10 +1,33 @@
 load("//rules/typescript:index.bzl", "ts_library")
+load(":defs.bzl", "TERMINAL_V2_ENABLED")
 
 package(default_visibility = ["//visibility:public"])
 
+exports_files(glob(["*.css"]))
+
+# TODO(bduffany): Once we are on terminal v2, remove all these
+# targets except for terminal2, and rename terminal2 to "terminal".
+genrule(
+    name = "index",
+    outs = ["terminal.tsx"],
+    cmd_bash = """
+        echo '
+import { TerminalComponent } from "./%s";
+
+export { TerminalComponent };
+export default TerminalComponent;' > $@
+    """ % ("terminal2" if TERMINAL_V2_ENABLED else "terminal1"),
```

Comment:
I'd rather just include both terminals and have larger js bundles for a bit during the transition.

Reply:
Resolved offline, will keep this as-is.